app-samples/Cognitive-Services-Face-WPF/Sample-WPF/Controls/UIHelper.cs

```diff
@@ -130,13 +130,45 @@ public static IEnumerable<Face> CalculateFaceRectangleForRendering(IList<Detecte
 
             foreach (var face in faces)
             {
+                var Left = (int)((face.FaceRectangle.Left * scale) + uiXOffset);
+                var Top = (int)((face.FaceRectangle.Top * scale) + uiYOffset);
+
+                // FaceAngle used for rotating face rectangles, default value is 0 (not rotated).
+                double FaceAngle = 0;
+
+                // If there has headpose attributes, will re-calculate the left/top(X/Y) position.
```

Comment:
`If head pose attributes have been obtained, re-calculate the left & top (X & Y) positions.`

Reply:
Done.